packages/appium/lib/cli/parser-helpers.js

```diff
@@ -58,7 +58,7 @@ function parsePluginNames (names) {
   }
 }
 
-function parseDefaultCaps (capsOrPath) {
+function parseJsonStringOrFile (capsOrPath) {
```

Comment:
The error message here is confusing for driverArgs, since it says about capabilities. We should either have a possibility to customize it via arguments or make it neutral